src/main/java/de/retest/recheck/review/ignore/AttributeShouldIgnore.java

```diff
@@ -0,0 +1,57 @@
+package de.retest.recheck.review.ignore;
+
+import java.util.regex.MatchResult;
+import java.util.regex.Pattern;
+
+import de.retest.recheck.ignore.ShouldIgnore;
+import de.retest.recheck.review.ignore.io.RegexLoader;
+import de.retest.recheck.ui.descriptors.Element;
+import de.retest.recheck.ui.diff.AttributeDifference;
+
+public class AttributeShouldIgnore implements ShouldIgnore {
+
+	private final String attribute;
+
+	public AttributeShouldIgnore( final String attribute ) {
+		this.attribute = attribute;
+	}
+
+	@Override
+	public boolean shouldIgnoreElement( final Element element ) {
+		return false;
+	}
+
+	@Override
+	public boolean shouldIgnoreAttributeDifference( final Element element,
+			final AttributeDifference attributeDifference ) {
+		return attributeDifference.getKey().equals( attribute );
+	}
+
+	@Override
+	public String toString() {
+		return String.format( AttributeShouldIgnoreLoader.FORMAT, attribute );
+	}
+
+	public static class AttributeShouldIgnoreLoader extends RegexLoader<AttributeShouldIgnore> {
+
+		private static final String MATCHER = "matcher: ";
```

Comment:
`:` refers to a delegate, e.g. if this matcher delegates to an other matcher, whereas `=` refers to a variable. Thus this should be "attribute=%s".

Reply:
That meant that matcher should be removed and only `attribute=outline` should be left, since now, your test could use `ElementShouldIgnore` instead of `AttributeShouldIgnore` since the regex matches, but will fail to find a matcher for `key=outline`.